### PR TITLE
Fix memory leaks

### PR DESCRIPTION
**Pull request by sankhesh**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -246,7 +269,8 @@ void demoRDV()
 
     RdvReader reader;
 
-    auto polyData = vtkPolyData::New();
+    auto polyData =
```
Comment (aashish24) [inline src/main.cpp:272]: @sankhesh do we need to use auto?

Comment (aashish24) [reply src/main.cpp:272]: meaning is there a specific reason, I am just thinking from the best practices of VTK point of view.

**Review by aashish24 — APPROVED** [review]
LGTM just had one style question.

File: src/main.cpp (hunk 1 above)
Comment (sankhesh) [reply src/main.cpp:272]: VTK does not have a strict convention (yet, anyway) of whether or not to use `auto`. 

Personally, I prefer **not** using it for code readability. In this case, it is your choice.

**Comment by aashish24** [comment]
> Personally, I prefer not using it for code readability. In this case, it is your choice.

Let's take it out in another PR for now this is good.

**Pull request merged**